src/components/FeatureViews/AdminPortal/AdminDashboard.js

```diff
@@ -229,47 +242,55 @@ const AdminDashboard = (props) => {
     ],
   }
 
+  const getTime = (time) => {
+    const minutes = parseInt(time.slice(-2), 10) / 60;
+    const hours = parseInt(time.substring(0,2), 10);
+    return hours + minutes;
+  }; 
+  
+   const unpaidSessionsObj = () => {
+     
+       const parentObj = data?.unpaidSessions.map(user=> {
+       const { endTime, startTime, hourlyTuition } = user.course
+       const totalTime = getTime(endTime) - getTime(startTime);
+       const dollarsPerSession = totalTime * hourlyTuition;
+       const lastPaidSessionDateTime = user.lastPaidSessionDatetime.substring(0,10).replace("-", "").replace("-", "");
+       const missedPaymentSessions = moment().diff(lastPaidSessionDateTime, 'days') / 7
+       const amountDue = dollarsPerSession * Math.ceil(missedPaymentSessions)
+       const firstName= user.paymentList[0].parent.user.firstName;
+       const lastName= user.paymentList[0].parent.user.lastName;
+       const id= user.paymentList[0].parent.user.id;
+       return {name: `${firstName} ${lastName}`, initial: `${firstName.charAt(0)}${lastName.charAt(0)}`, id: id, due: amountDue}
+    });
+    return parentObj || [];
+   };
+
+  const filterAndAddDuplicates = arr => {
```

Comment:
Can you put some documentation above this line to describe what you're doing here? You have your comments below, you can put that in the documentation. This is an example of documentation/how you should format it:
```
/**
 * Pads a number to the desired length, filling with leading zeros
 * @param {Number} integer Number to pad
 * @param {String} length Minimum number of digits
 * @returns {String} Padded integer
 */
```